README.md

```diff
@@ -6,6 +6,10 @@
 * Mohammad_Murtuza
 * Chenyu
 * Wenyu Zheng
+<<<<<<< HEAD
+* Kayla Laufer
+=======
 * Safal Gautam
 * Andi
 * Kayla
+>>>>>>> a5c63c5d069b143519de1ffc350779104ab7de8d
```

Comment:
Resolve any conflicts by comparing the code between `<<<<<<< HEAD` and `=======` vs the code between `=======` and `>>>>>>> a5c63c5d069b143519de1ffc350779104ab7de8d`. Delete/add as needed. For example here, I would delete the `Kayla` at the bottom, then delete the conflict markers (<<<, ===, >>>). The final file would look like:

```
* Mohammad_Murtuza
* Chenyu
* Wenyu Zhengt
* Kayla Laufer
* Safal Gautam
* Andi